adventure/api.py

```diff
@@ -63,5 +63,16 @@ def move(request):
 @csrf_exempt
 @api_view(["POST"])
 def say(request):
-    # IMPLEMENT
-    return JsonResponse({'error':"Not yet implemented"}, safe=True, status=500)
+    player = request.user.player
+    player_name = player.user.username
+    player_uuid = player.uuid
+    player_id = player.id
+    data = json.loads(request.body)
+    message = data['message']
```

Comment:
It's a small thing, but have you thought about how you might handle the case where a user submits a blank message or a malicious code? It's definitely stretch but it could interesting to think about.